### DISCUSSION TITLE
How should I solve this error?

### DISCUSSION THREAD

**Discussion by mrbardia72** [golangci/golangci-lint#3433] [Q&A] [answered]
```
ERRO Running error: 1 error occurred:
        * can't run linter goanalysis_metalinter: goanalysis_metalinter: buildir: package "atomic" (isInitialPkg: false, needAnalyzeSource: true): T
```

**Comment by ldez** [comment]
Hello,

It's a compilation error inside your code, check your code and run a `go build`.

**Reply by mrbardia72** [reply]
I ran the following command for the build, the output was OK
`go build -v -o $(ROOT)/chaincode -ldflags="-s -w" $(ROOT)/*.go`

Again it shows the same error
golangci-lint run -v

**Reply by ldez** [reply]
`buildir` is related to the compilation check, so there is a compilation issue.

When I see `$(ROOT)/*.go`, I think you are trying to do something unusual.
First, I think you are trying to build outside of a Go module.
And then I don't know what you are trying to do with `*.go`, commonly we just use a package path.

So I think if you just run `go build` (without any arguments), at the same place where you are trying to run golangci-lint, you will see the problem.

**Reply by mrbardia72** [reply]
I just hit `go build` but still the same error

**Reply by mrbardia72** [reply, marked as answer]
@ldez 
Adding in the config file solved the problem
```
run:
    go: 1.18
```
